docs/development/developer_guide.md

```diff
@@ -57,15 +57,18 @@ export KUBEFLOW_NAMESPACE=$(your_namespace)
 After the cluster is up, the TFJob CRD should be created on the cluster.
 
 ```bash
-kubectl create -f ./examples/crd/crd-v1.yaml
+kubectl create -f ./manifests/base/kubeflow.org_mxjobs.yaml
+kubectl create -f ./manifests/base/kubeflow.org_pytorchjobs.yaml
+kubectl create -f ./manifests/base/kubeflow.org_tfjobs.yaml
+kubectl create -f ./manifests/base/kubeflow.org_xgboostjobs.yaml
```

Comment:
I think we can use [`make install`](https://github.com/kubeflow/tf-operator/blob/master/Makefile#L74) here.

Reply:
done.